src/OpenCV_Mat.jl

```diff
@@ -41,27 +41,35 @@ inline void ptr_val3(cv::Mat &img, int row, int col, std::vector<double> vec)
 
 function pixget(img, row::Int, col::Int)
   (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
-
-  # Grayscale and binary images
-  val = @cxx ptr_val(img, row, col);
-  println(val)
-  return (val)
-
-  # RGB images
-  vec = @cxx ptr_val3(img, row, col);
-  println([int(at(vec, 0)), int(at(vec, 1)), int(at(vec, 2))])
-  return (vec)
+  image = Images.separate(img);
```

Comment:
Points worth keeping in mind:
- `separate` and `colordim` are deprecated
- ImageCore.jl is a much smaller dependency
- In modern Images.jl (which is built on top of ImageCore), the color dimension always has to be placed first
- There's a `permuteddimsview` to lazily-permute the dimensions when that's not true.
- The various new views (`normedview`, `rawview`, `channelview`, `colorview`) will `reinterpret` a block of memory when it's feasible to do so, and otherwise return a wrapper that gives the same functionality. So if you use those functions, getting out an `Array` means that you can safely pass it directly to C++ without making copies.
- If you don't get out an `Array`, but get some kind of `AbstractArray`, beware that you probably can't pass such an array as a `Ptr` to C++ routines and expect it to work.